### PR TITLE
Responsive Desktop Version

### PR DESCRIPTION
**Pull request by D4nielJ**
## Hello Reviewer

This PR mostly implements CSS3 Media Queries for mid-size and wide screens to my portfolio

You can check the [Live Demo here.](https://d4nielj.github.io/developer-portfolio/)

Let me know any changes or suggestions.

Cheers,
Daniel Jaramillo

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -444,3 +451,174 @@ header {
...
+  }
+
+  #introduction {
+    background: #fff url(../media/header-bg.png) 50% 50% no-repeat;
+  }
+
+  #introduction p {
+    max-width: 80%;
+  }
+}
```
Comment (Davidosky007) [inline css/styles.css]: I suggest you stick to the design as much as possible, This is your design.
Please compare, check the difference between both decoration positions and make the necessary adjustment
Screenshot (33)

Figma design 
Screenshot (34)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @the-catalystmc @D4nielJ, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: css/styles.css (hunk 1 above)
Comment (D4nielJ) [reply css/styles.css]: Adjustments were made so it looks more like the original.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @D4nielJ @the-catalystmc 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

nice

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**